boards/imix/chip_layout.ld

```diff
@@ -1,8 +1,8 @@
 /* Memory Spaces Definitions, 448K flash, 64K ram */
 ROM_ORIGIN  = 0x00010000; /* Use bootloader starting at 0x0000 */
-ROM_LENGTH  = 0x00020000;
-PROG_ORIGIN = 0x00030000;
-PROG_LENGTH = 0x00040000;
+ROM_LENGTH  = 0x00025000;
```

Comment:
Sigh... this is kind of a bummer, but for sanity, I compiled open thread for the CC2650 and it looks like it's on the order of 86-105kB for _just_ open thread (obviously with no sensor drivers or USB or anything), and this is still under half of flash for what's a basically very featureful kernel.